ansible/vars/external_vars.yml

```diff
@@ -0,0 +1,2 @@
+---
+ADMIN_EMAIL: admin@zap.me
```

Comment:
`admin_email` var already passed in via `ansible_deploy.sh` better to remove this and use that which is already there

Reply:
changed the jinja2 file to look for {{ admin_email }}. testing to see if it works.

Reply:
removed the vars file, the entry for the vars file in the fail2ban.yml file and changed the jinja2 file to use {{ admin_email }} instead.